nbconvert/nbconvertapp.py

```diff
@@ -400,7 +419,7 @@ def convert_notebooks(self):
             self.exit(1)
         
         # initialize the exporter
-        self.exporter = exporter_map[self.export_format](config=self.config)
+        self.exporter = get_exporter(self.export_format)(config=self.config)
```

Comment:
@takluyver I think the double call here can be confusing and decrease readability. Thoughts ?